arangod/Aql/ExecutionBlockImpl.tpp

```diff
@@ -1237,6 +1252,8 @@ auto ExecutionBlockImpl<Executor>::shadowRowForwarding(AqlCallStack& stack)
       return ExecState::NEXTSUBQUERY;
     }
 
+    [[maybe_unused]] bool const hasDoneNothing =
```

Comment:
Why is this maybe_unused and the others not?
It is only used in one of the If cases but so are the ones above?

Reply:
an accident.
```suggestion
    bool const hasDoneNothing =
```